chromium_src/chrome/browser/extensions/api/cookies/cookies_api.cc

```diff
@@ -10,7 +10,7 @@
 // in upstream, we'll be able to manage Tor profiles like any other main-OTR
 // profile.
 
-#define IsSerializeable(...) IsSerializeable(__VA_ARGS__) && !profile_->IsTor()
+#define IsSerializeable(...) IsSerializeable(__VA_ARGS__) || profile_->IsTor()
```

Comment:
so this is not covered with a test currently? it it possible to do so?

Reply:
Yes, it's hard to do